pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -19,10 +19,10 @@ rec {
   # Policy: use the highest stable version as the default (on our master).
```

Comment:
This package has no maintainers, please consider adding yourself to the maintainer list if you can help out.